content/docs/atom-guide/dev-guide/shaders/shader-build-pipeline.md

```diff
@@ -21,7 +21,7 @@ Shaders are made up of several files:
 
 There are a few key extensions of AZSL, which affect the shader build pipeline:
 - **Shader Resource Groups (SRGs)**: A container for application visible variables, which are defined in code with the class `ShaderResourceGroup`. SRGs can be defined in `.azsl` files as *partial SRGs*, which define only a portion of an SRG. You can combine the partial SRGs in a single `.srgi` file.<!-- Learn more about SRGs in the [AZSL Reference](azsl/reference/).DRAFT TOPIC-->
```

Comment:
* Remove comment about a reference to a page. We have these docs now :) 
```suggestion
- **Shader Resource Groups (SRGs)**: A container for application visible variables, which are defined in code with the class `ShaderResourceGroup`. SRGs can be defined in `.azsl` files as *partial SRGs*, which define only a portion of an SRG. You can combine the partial SRGs in a single `.srgi` file.

  Learn more about [SRGs](/docs/atom-guide/dev-guide/shaders/azsl/#shader-resource-groups) in the AZSL section.
```

Reply:
Fixed.